plutus-ledger-api/test/Spec/Eval.hs

```diff
@@ -9,13 +9,14 @@ import PlutusCore.MkPlc
 import PlutusCore.StdLib.Data.Unit
 import PlutusCore.Version as PLC
 import PlutusLedgerApi.Common.Versions
-import PlutusLedgerApi.Test.EvaluationContext (evalCtxForTesting)
+import PlutusLedgerApi.Test.EvaluationContext
```

Comment:
You could still say what you're importing here (I think just `v1_costModelParamsForTesting`). I find that quite helpful (I won't demand that you do that though).